Data_Structures_Answers.md

```diff
@@ -1,18 +1,22 @@
 Add your answers to the questions below.
 
 1. What is the runtime complexity of your ring buffer's `append` method?
+Only one if statment O(1)
 
 2. What is the space complexity of your ring buffer's `append` function?
+    O(n)
```

Comment:
should be o(1) also since you aren't allocating any additional memory when appending.

Reply:
fixed